io/index.js

```diff
@@ -15,20 +16,98 @@ async function retrieveGraphData(){
   const {numberOfDataPoints, dataPoints} = await loadJson('./data.json');
   console.log('numberOfDataPoints', numberOfDataPoints);
   console.log('dataPoints', dataPoints);
-  
+
   return dataPoints;
 }
 
 
+/**
+ * Function that allows usr to add new data points to current
+ * json data
+ */
+function getNewDataPoint(){
+  // A draft of questions
+  var monthNames = ["JAN", "FEB", "MAR", "APR", "MAY", "JUNE", "JULY", "AUG", "SEP", "OCT", "NOV", "DEC"];
+  var questions = [
+    {
+      type: 'input',
+      name: 'value',
+      message: "What is the next value?", 
+	  validate: function(value) {
+		  if (parseFloat(value.trim()) > 0 && parseFloat(value.trim()) < 1) {
+			return true;
+		}
+
+		return 'Please enter a value between 0 and 1'
+	  }
+      // A validation will be needed here to ensure this value is a
+      // float between 0 and 1
+    },
+    {
+      type: 'input',
+      name: 'label',
+      message: "What is the label for this (next) value?",
+      default: function() {
+        // I need to adjust this to local time
+        var today = new Date();
+		// Year may be important?
+        var month = monthNames[today.getUTCMonth()]
+	    var day = today.getUTCDay()
+
+
+        return month + " " + day;
+	  },
+	  validate: function(value) {
+		var split = value.split(" ");
+		if (!monthNames.includes(split[0].trim().toUpperCase())) {
+	      return "Choose a month in: " + monthNames.join(", ");
+		} else {
+			day_input = parseInt(split[1].trim());
+			if (isNaN(day_input) || day_input < 0 || day_input > 31) {
```

Comment:
```suggestion
			if (isNaN(day_input) || day_input < 1 || day_input > 31) {
```
Lol can't have a day 0.